config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  root 'application#hello'
+  root 'login#login'
+  get '/home', to: 'home#home'
+  get '/register', to: 'users#new'
+  post '/register',  to: 'users#create'
+  get '/login', to: 'login#login'
```

Comment:
Ohh yeah, didn't know you changed `index` to `login`. I would argue that it would be better to keep with a RESTful implementation and switch `login` back to `index`, but DWYTIB. 

http://guides.rubyonrails.org/routing.html
Look at 2.2